source/_integrations/vicare.markdown

```diff
@@ -24,51 +25,88 @@ ha_codeowners:
+Represents the domestic hot water controls of your device.
 
-## Viessmann API limits
+{% note %}
+It is not possible to turn on/off water heating via the water heater {% term integration %} since this would conflict with the operation modes of the heating integration. Therefore, the operation mode of that integration is just available as an attribute and cannot be modified.
+{% endnote %}
 
-The Viessmann API is rate-limited. If you exceed one of the limits below you will be banned for 24 hours:
+### Sensor
 
-- Limit 1: 120 calls for a time window of 10 minutes
-- Limit 2: 1450 calls for a time window of 24 hours
+Additional data for a device is available as separate sensors. The sensors are automatically discovered based on the available API data points.
 
-The {% term integration %} polls the Viessmann API every 60 seconds and will work within these limits. Note however that any additional requests to the API, e.g., by setting the temperature via the {% term integration %} but also by interacting with the ViCare app also counts into those limits.
+### Button
 
-## Climate
+Button entities are available for triggering like a one-time charge of the water heater.
+
+### Number
 
-A note about the current temperature attribute: Viessmann devices with room temperature sensing will show the current room temperature via `current_temperature`. All other devices will show the current supply temperature of the heating circuit.
+Number entities are available to adjust values like the predefined temperature for different heating programs or the heating curve shift and slope.
 
-### Supported services `climate.vicare_heating`
+## Services
+
+The following services of the [climate integration](/integrations/climate/) are provided by the ViCare integration: `set_temperature`, `set_hvac_mode`, `set_preset_mode` 
+
+The following services of the [water_heater integration](/integrations/water_heater/) are provided by the ViCare integration: `set_temperature`
+
+### Service `vicare.set_vicare_mode`
+
+Set the mode for the climate device as defined by Viessmann (see [set_hvac_mode](#service-climateset_hvac_mode) for a mapping to Home Assistant Climate modes. This allows more-fine grained control of the heating modes.
 
-The following services of the [Climate integration](/integrations/climate/) are provided by the ViCare {% term integration %}: `set_temperature`, `set_hvac_mode`, `set_preset_mode` 
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id` | yes | String or list of strings that point at `entity_id`'s of climate devices to control. To target all entities, use the `all` keyword instead of entity_id.
+| `vicare_mode` | no | New value of ViCare mode. For supported values, see the `vicare_modes` attribute of the climate {% term entity %}.
```

Comment:
This seems to break the table.
```suggestion
| `vicare_mode` | no | New value of ViCare mode. For supported values, see the `vicare_modes` attribute of the climate entity.
```

<img width="742" alt="Bildschirmfoto 2024-07-31 um 14 02 43" src="https://github.com/user-attachments/assets/17ac0215-55e9-4a94-ae2f-eb6cec021d6c">

should be 

<img width="268" alt="Bildschirmfoto 2024-07-31 um 14 02 31" src="https://github.com/user-attachments/assets/fc653bcf-3adf-49f1-ac17-1255a551a71d">

Reply:
I already removed the terminlogy tag here to avoid the issue. I guess it's no necessary to highlight each and every term, right?

Reply:
no, but it is now my test case https://github.com/home-assistant/home-assistant.io/pull/34049